conan/tools/cmake/cmakedeps/templates/target_data.py

```diff
@@ -66,6 +66,7 @@ def template(self):
               set({{ pkg_name }}_DEFINITIONS{{ config_suffix }} {{ global_cpp.defines }})
               set({{ pkg_name }}_SHARED_LINK_FLAGS{{ config_suffix }} {{ global_cpp.sharedlinkflags_list }})
               set({{ pkg_name }}_EXE_LINK_FLAGS{{ config_suffix }} {{ global_cpp.exelinkflags_list }})
+              set({{ pkg_name }}_OBJECTS{{ config_suffix }} {{ global_cpp.objects_list }})
```

Comment:
Only global? The cmake target_configuration seems to try to apply per-component ones, but seems per-component not defined here?